dllNetwork/server/ServerI2CInstruction.C

```diff
@@ -80,6 +82,88 @@ void system_iic_close(system_iic * handle)
     }
 }
 
+uint32_t system_iic_reset(Handle * i_handle, int i_type)
+{
+    int rc = 0;
+
+    if( i_type == ADAL_RESET_LIGHT || i_type == ADAL_RESET_FULL )
+    {
+        rc = adal_iic_set_register( (adal_t *) i_handle, ADAL_IIC_REG_RESET, 0x0 );
+        if ( global_server_debug )
+            printf("adal_iic_set_register( (adal_t *) i_handle, ADAL_IIC_REG_RESET, 0x0 )  rc(%d) errno(%d)\n", rc, errno);
+        if ( rc < 0 )
+            return rc;
+    }
+
+    // always attempt full reset even for light mode
+    if( i_type == ADAL_RESET_FULL || i_type == ADAL_RESET_LIGHT)
+    {
+        // set diag_mode on
+#ifdef _BIG_ENDIAN
```

Comment:
what is going on with this big endian check?

Reply:
This was the only time we were writing data, other than 0's.  Was just used to doing big endian checking for this stuff so included it.   Do we only need the little endian version here and can remove the big endian stuff?